From 4d37b60c81cf76f1d1e2f9cde5ee33e052593d54 Mon Sep 17 00:00:00 2001
From: Scott Little <scott.little@windriver.com>
Date: Mon, 2 Oct 2017 16:45:32 -0400
Subject: [PATCH 1/3] WRS:
 spec-wdmd-Kernel-watchdog-enable-and-reboot-on-error.patch

---
 SPECS/sanlock.spec | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/SPECS/sanlock.spec b/SPECS/sanlock.spec
index 18ce7dc..92b81bb 100644
--- a/SPECS/sanlock.spec
+++ b/SPECS/sanlock.spec
@@ -30,6 +30,7 @@ Requires(postun): systemd-units
 Source0:        https://releases.pagure.org/sanlock/%{name}-%{version}.tar.gz
 
 # Patch0: 0001-foo.patch
+Patch100: wdmd-Kernel-watchdog-enable-and-reboot-on-error.patch
 
 %description
 The sanlock daemon manages leases for applications on hosts using shared storage.
@@ -37,6 +38,7 @@ The sanlock daemon manages leases for applications on hosts using shared storage
 %prep
 %setup -q
 # %patch0 -p1 -b .0001-foo.patch
+%patch100 -p1 -b .disable-wdmd-daemon-on-boot
 
 %build
 # upstream does not require configure
-- 
1.9.1

